{"uuid":"9b16f077-8476-4a17-871f-76b59670a11e","name":"An uncaught error was detected outside of a test","historyId":"f08906e77502c2fa5af44d0e7cc5018d:d41d8cd98f00b204e9800998ecf8427e","status":"broken","statusDetails":{"message":"The following error originated from your test code, not from Cypress.\n\n  > dataJson is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","trace":"ReferenceError: The following error originated from your test code, not from Cypress.\n\n  > dataJson is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at eval (webpack://cypress2/./cypress/e2e/API testing/post3.cy.js:1:8)\n    at eval (http://localhost:54491/__cypress/tests?p=cypress\\e2e\\API%20testing\\post3.cy.js:80:3)\n    at eval (http://localhost:54491/__cypress/tests?p=cypress\\e2e\\API%20testing\\post3.cy.js:82:12)\n    at eval (<anonymous>)\nFrom previous event:\n    at Promise.longStackTracesCaptureStackTrace [as _captureStackTrace] (http://localhost:54491/__cypress/runner/cypress_runner.js:3486:19)\n    at Promise._then (http://localhost:54491/__cypress/runner/cypress_runner.js:1239:17)\n    at Promise.then (http://localhost:54491/__cypress/runner/cypress_runner.js:1132:17)\n    at runScriptsFromUrls (http://localhost:54491/__cypress/runner/cypress_runner.js:111657:136)\n    at Object.runScripts (http://localhost:54491/__cypress/runner/cypress_runner.js:111698:12)\n    at $Cypress.onSpecWindow (http://localhost:54491/__cypress/runner/cypress_runner.js:41308:67)"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"cypress"},{"name":"host","value":"ASCINLAP60907"},{"name":"thread","value":"pid-20624-worker-main"},{"name":"package","value":"cypress.e2e.API testing.post3.cy.js"}],"links":[],"start":1731566032550,"fullName":"cypress/e2e/API testing/post3.cy.js#An uncaught error was detected outside of a test","testCaseId":"f08906e77502c2fa5af44d0e7cc5018d","stop":1731566032606}